src/bindgen/ir/enumeration.rs

```diff
@@ -250,8 +252,12 @@ impl Enum {
 
     pub fn load(item: &syn::ItemEnum, mod_cfg: Option<&Cfg>) -> Result<Enum, String> {
         let repr = Repr::load(&item.attrs)?;
-        if repr == Repr::RUST {
-            return Err("Enum not marked with a valid repr(prim) or repr(C).".to_owned());
+        if repr.style == ReprStyle::Rust && repr.ty.is_none() {
```

Comment:
Why this change? I don't think `#[repr(align = ..)] enum { ... }` has a stable ABI.

Reply:
`#[repr(align(x))] enum` still results in an opaque type -- the reason for this change was that I dropped the comparisons to `Repr` because a few of them caused tests to fail once I added `Repr.align` (because now `repr == Repr::RUST` fails if the type has alignments set).

I can re-add it back, but I personally find it more confusing to have two different ways of effectively checking `repr.style`.

Reply:
Also, `#[repr(align(x))] enum` was stabilised in Rust 1.37 -- https://github.com/rust-lang/rust/pull/61229.

Reply:
Oh, this was my mistake. What I meant is, why isn't this just `if repr.style == ReprStyle::Rust { return Err(..); }`. But this is wrong as we'd stop generating `#[repr(u8)]` etc.

So please disregard this :)

Reply:
Yup, that was the reason for the removal of direct `repr ==` comparisons.